packages/@aws-cdk/aws-scheduler-targets-alpha/lib/target.ts

```diff
@@ -148,25 +148,14 @@ export abstract class ScheduleTargetBase {
   }
 
   /**
-   * Allow a schedule to send events with failed invocation to an Amazon SQS queue.
-   * @param schedule schedule to add DLQ to
-   * @param queue the DLQ
+   * Allow schedule to send events with failed invocation to an Amazon SQS queue.
    */
-  private addToDeadLetterQueueResourcePolicy(schedule: ISchedule, queue: sqs.IQueue) {
-    if (!sameEnvDimension(schedule.env.region, queue.env.region)) {
-      throw new Error(`Cannot assign Dead Letter Queue in region ${queue.env.region} to the schedule ${Names.nodeUniqueId(schedule.node)} in region ${schedule.env.region}. Both the queue and the schedule must be in the same region.`);
-    }
-
+  private addDeadLetterQueueActionToRole(schedule: ISchedule, role: iam.IRole, queue: sqs.IQueue) {
     // Skip Resource Policy creation if the Queue is not in the same account.
     // There is no way to add a target onto an imported schedule, so we can assume we will run the following code only
     // in the account where the schedule is created.
     if (sameEnvDimension(schedule.env.account, queue.env.account)) {
```

Comment:
This check is not needed. The queue and the schedule can be in different account and in such case, we still want to add the policy to the role.

Reply:
Ah I see, will remove